### PR TITLE
cassidoo Feb 3rd 2025

### PR DESCRIPTION
**Pull request by Barbacoa08**
evaluate postfix equation

## Summary by CodeRabbit

- **New Features**
	- Introduced functionality to evaluate postfix expressions using basic arithmetic operations.
- **Documentation**
	- Added guidelines and examples outlining the expected behavior of postfix evaluation.
- **Tests**
	- Implemented comprehensive test suites to validate evaluation logic, including handling of edge cases and proper order of operations.
- **Chores**
	- Updated build and test scripts to incorporate the new postfix evaluation functionality.

### CODE REVIEW

File: src/evaluate-postfix/solution/evaluate-postfix.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	if (s.length < 3) return 0;
+
+	const operations: string[] = [];
+
+	for (const element of s) {
+		if (+element) {
+			operations.push(element);
+		} else {
+			const a = operations.shift();
+			const b = operations.shift();
```
[the remark this answers is not in the document]
Comment (Barbacoa08) [reply src/evaluate-postfix/solution/evaluate-postfix.ts:11]: updated, thanks

File: src/evaluate-postfix/solution/evaluate-postfix.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+			// biome-ignore lint/security/noGlobalEval: `eval` is the only way to compute
+			const result = eval(`${a} ${element} ${b}`);
```
[the remark this answers is not in the document]
Comment (Barbacoa08) [reply src/evaluate-postfix/solution/evaluate-postfix.ts:14]: updated, thanks
